packages/three-vrm-core/src/VRMCoreParameters.ts

```diff
@@ -1,3 +1,4 @@
+import type * as THREE from 'three';
```

Comment:
This can also be done as `import { type Group } from 'three';` but requires more changes overall. I'm ok with current approach.